### DISCUSSION TITLE
Have to click twice on item to get it to change

### DISCUSSION THREAD

**Discussion by ghost** [react-hook-form/react-hook-form#5565] [Q&A] [answered]
When loading the initial value of react-select, and wanting to change the option, I have to click on an item twice for it to change. If I click on something the first time, the initial loaded option remains.

Code sandbox

https://codesandbox.io/s/great-swartz-wwycy?file=/src/App.js

It actually does the same on your example if I change the react-hook-form version to 7.6.1

https://codesandbox.io/s/react-hook-form-v7-controller-forked-0l2pm

**Comment by bluebill1049** [comment, marked as answer]
use the latest version: https://codesandbox.io/s/quizzical-pasteur-3h7bh?file=/src/App.js
